view/frontend/layout/hyva_catalogsearch_result_index.xml

```diff
@@ -31,12 +31,12 @@
         <referenceContainer name="sidebar.main">
             <referenceBlock name="catalogsearch.leftnav"
                    template="Shellpea_AdvancedElasticsuiteCatalog::catalog/layer/view.phtml">
-                <referenceBlock name="catalogsearch.navigation.state" />
+                <referenceBlock name="catalogsearch.navigation.state" template="Shellpea_AdvancedElasticsuiteCatalog::catalog/layer/state.phtml" />
                 <referenceBlock name="catalogsearch.navigation.renderer">
 
                     <referenceBlock name="catalogsearch.navigation.renderer.attribute"
                         template="Shellpea_AdvancedElasticsuiteCatalog::catalog/layer/filter/attribute.phtml">
-                        <referenceBlock name="attribute-filter-js" remove="true"/>
+                        <referenceBlock name="attribute-filter-js"/>
```

Comment:
As we're not doing anything to the attribute-filter-js block, let's remove this line from the layout

```suggestion
```

Reply:
One of the reasons why there was a problem on the search page is that this block is needed in the current version of the module😉